package.json

```diff
@@ -70,7 +68,9 @@
     "mockery": "^2.1.0",
     "nyc": "^11.0.3",
     "optimize-js-plugin": "0.0.4",
+    "react": "^15.3.0",
```

Comment:
can we have both `react` and `react-dom` use `^16.0.0` instead of `^15.3.0`? it is usable back to `15.3.0` but is built with latest first and foremost. also, with this setup the tests would fail because they are built assuming react 16 for development (hence `enzyme-adapter-react-16`)